geode-core/src/distributedTest/java/org/apache/geode/internal/cache/PartitionedRegionSingleHopDUnitTest.java

```diff
@@ -1405,7 +1385,8 @@ private void waitForLocalBucketsCreation() {
     await().untilAsserted(() -> assertThat(pr.getDataStore().getAllLocalBuckets()).hasSize(4));
```

Comment:
While you're cleaning up the test code, warnings on lines 1328, 1333, 1338 and 1342 can be fixed by using `PutFunction<>()`.